src/main/resources/application-mysql.properties

```diff
@@ -0,0 +1,4 @@
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
The main thing we're doing here is using Spring profile configurations (https://docs.spring.io/spring-boot/docs/current/reference/html/howto-properties-and-configuration.html), as opposed to the previous methodology fo copying in app properties based on an environment flag.

We have also changed config syntax to use environment variables for the actual MySQL connection, which will need to be coordinated with an update to the docker compose setup in kotlin-dev-env.

Reply:
This wasn't really part of this ticket, but somehow this update got wrapped up in this branch...